driver-async/src/test/functional/com/mongodb/async/client/MongoClientSessionSpecification.groovy

```diff
@@ -311,12 +311,14 @@ class MongoClientSessionSpecification extends FunctionalSpecification {
     @IgnoreIf({ !serverVersionAtLeast(4, 0) || !isDiscoverableReplicaSet() })
     def 'should ignore unacknowledged write concern when in a transaction'() {
         given:
+        def collection = getMongoClient().getDatabase(getDatabaseName()).getCollection(getCollectionName())
```

Comment:
This test was incorrectly passing due to JAVA-3282.  Now that it's fixed and MixedBulkWriteOperation is fully ignoring the write concern on the collection, the test is failing because you can't create a collection implicitly inside a transaction.  Fixing it by ensuring that the collection is created before the transaction is started.